### DISCUSSION TITLE
How can I export a google map which already have got 30 strm DEM as a TIFF in blender 3.1?

### DISCUSSION THREAD

**Discussion by Hareden** [domlysz/BlenderGIS#674] [General]
Just a small question ,I`m a rookie!
<img width="519" alt="xy" src="https://user-images.githubusercontent.com/106158319/170799843-376eaebd-806b-47ad-8b21-1623f744ceea.png">
Firstly I pressed  the “E” and catch a local place via blendergis google map.
Secondly I got the elevation via the blendergis sram 30m.
<img width="235" alt="get dem" src="https://user-images.githubusercontent.com/106158319/170800028-47ce8a63-ff6d-4183-b713-130448931cdb.png">
<img width="487" alt="xyz" src="https://user-images.githubusercontent.com/106158319/170800030-e54385bd-9357-4c9f-8d9f-6e1716b673b5.png">
Finally I want to export the map which got the dem as a TIFF.
or I just aimmed to get the DEM in XY dimension like this.
<img width="410" alt="dem xy" src="https://user-images.githubusercontent.com/106158319/170800230-c2970c03-a903-42e3-96bb-23dd73ca5b05.png">
How should I do?

Looking forward to your favourable reply.

**Comment by domlysz** [comment]
If your blend file is already saved, the TIFF will be putted into the same folder
